src/audio/AudioResource.ts

```diff
@@ -189,6 +189,21 @@ export function inferStreamType(stream: Readable): {
 	return { streamType: StreamType.Arbitrary, hasVolume: false };
 }
 
+/**
+ * Creates an audio resource that can be played be audio players.
```

Comment:
Creates an audio resource that can be played **by** audio players.

Reply:
I just fixed that, thanks for picking that up!